src/Interpreters/JoinedTables.cpp

```diff
@@ -201,7 +188,22 @@ StoragePtr JoinedTables::getLeftTableStorage()
 bool JoinedTables::resolveTables()
 {
     tables_with_columns = getDatabaseAndTablesWithColumns(table_expressions, context);
-    checkTablesWithColumns(tables_with_columns, context);
+    assert(tables_with_columns.size == table_expressions.size);
```

Comment:
```
2020-05-20 20:04:29 /build/obj-x86_64-linux-gnu/../src/Interpreters/JoinedTables.cpp:191:32: error: reference to non-static member function must be called; did you mean to call it with no arguments? [clang-diagnostic-error]
2020-05-20 20:04:29     assert(tables_with_columns.size == table_expressions.size);
2020-05-20 20:04:29                                ^
2020-05-20 20:04:29 ../contrib/libc-headers/assert.h:93:27: note: expanded from macro 'assert'
```
If the size was different, but the assertion didn't fail, how long would it have taken me to notice?... Thank you, clang-tidy.